README.md

```diff
@@ -19,115 +19,52 @@
-To use these, you must be on Linux and have a conda environment with Python 3.9 or 3.10, CUDA 11.3 or 11.6, and PyTorch 1.12.1.
-You can install the latest with
-
-  ```bash
-  conda install xformers -c xformers/label/dev
-  ```
-
-</p></details>
-
-<details><summary> Build from source (dev mode) </summary><p>
-
-  These commands will fetch the latest version of the code and then install xFormers from source.
-  If you want to build the sparse attention CUDA kernels, please make sure that the next point is covered prior to running these instructions.
-
-  ```bash
-  git clone git@github.com:facebookresearch/xformers.git
-  git submodule update --init --recursive
-  conda create --name xformer_env python=3.8
-  conda activate xformer_env
-  cd xformers
-  pip install -r requirements.txt
-  pip install -e .
-  # or, for OSX
-  MACOSX_DEPLOYMENT_TARGET=10.9 CC=clang CXX=clang++ pip install -e .
-  ```
-
-</p></details>
-
-### Installing custom (non-pytorch) parts
+* **From source**: Alternatively, if no binaries are available (for instance for windows), you can also install from source:
 
-<details><summary> Sparse attention kernels </summary><p>
-
-Installing the CUDA-based sparse attention kernels may require extra care, as this mobilizes the CUDA toolchain. As a reminder, these kernels are built when you run `pip install -e .` and the CUDA buildchain is available (NVCC compiler). Re-building can for instance be done via `python3 setup.py clean && python3 setup.py develop`, so similarly wipe the `build` folder and redo a pip install -e.
-
-Some advices related to building these CUDA-specific components, tentatively adressing common pitfalls. Please make sure that:
-
-* NVCC and the current CUDA runtime match. Depending on your setup, you may be able to change the CUDA runtime with `module unload cuda module load cuda/xx.x`, possibly also `nvcc`
-* the version of GCC that you're using matches the current NVCC capabilities
-* the `TORCH_CUDA_ARCH_LIST` env variable is set to the architures that you want to support. A suggested setup (slow to build but comprehensive) is `export TORCH_CUDA_ARCH_LIST="6.0;6.1;6.2;7.0;7.2;8.0;8.6"`
-
-</p></details>
-
-<details><summary> Triton </summary><p>
-
-Some parts of xFormers use [Triton](http://www.triton-lang.org), and will only expose themselves if Triton is installed, and a compatible GPU is present (nVidia GPU with tensor cores). If Triton was not installed as part of the testing procedure, you can install it directly by running `pip install triton`. You can optionally test that the installation is successful by running one of the Triton-related benchmarks, for instance `python3 xformers/benchmarks/benchmark_triton_softmax.py`
-
-Triton will cache the compiled kernels to `/tmp/triton` by default. If this becomes an issue, this path can be specified through the `TRITON_CACHE_DIR` environment variable.
-
-</p></details>
+```bash
+# (Optional) Makes the build much faster
```

Comment:
maybe a word on memory consumption for some machines ? I kind of remember people getting OOMs there (though I would definitely recommend ninja of course)

Reply:
Added that to the troubleshooting section - thanks !